.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - 6
-
+  - "6"
```

Comment:
@bebraw I noticed on the official documentation for travis that they were using the version numbers as strings. Do I think this makes a difference, I am not 100% sure. But it appears that the tests are running against v6 now.